src/Microsoft.Data/DataFrameTable.cs

```diff
@@ -138,6 +138,76 @@ public void RemoveColumn(string columnName)
             }
         }
 
+        private void ResizeByOneAndAppend(BaseColumn column, object value)
+        {
+            long length = column.Length;
+            column.Resize(length + 1);
+            column[length] = value;
+        }
+
+        public void Append(IEnumerable<object> row)
+        {
+            IEnumerator<BaseColumn> columnEnumerator = _columns.GetEnumerator();
+            bool columnMoveNext = columnEnumerator.MoveNext();
+            if (row != null)
+            {
+                IEnumerator<object> rowEnumerator = row.GetEnumerator();
+                bool rowMoveNext = rowEnumerator.MoveNext();
+                while (columnMoveNext && rowMoveNext)
```

Comment:
What happens when I give too many values and there aren't enough columns? Do we just silently ignore that?

Reply:
Yup, they get ignored.

Reply:
Is that the best approach? It seems like a mistake by the user, and they would want to know about it.